### ISSUE TITLE
Tortuous lymphatic vessels

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7030]
**Preferred term label:**
Tortuous lymphatic vessels

**Synonyms**
Tortuous lymphatics

**Definition (free text, please give PubMed ID)**
The presence of an increased number of twists and turns of lymphatic vessels.

Comment: This feature can be ascertained by near-infrared fluorescence lymphatic imaging.

**Parent term (use hpo.jax.org/app)**
Abnormal lymphatic vessel morphology HP:0100766

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:619319
PMID: 26855770  1/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
